common/components/BalanceSidebar/TokenRow.jsx

```diff
@@ -0,0 +1,57 @@
+// @flow
+import React from 'react';
+import Big from 'big.js';
+import translate from 'translations';
+import { formatNumber } from 'utils/formatters';
+
+export default class TokenRow extends React.Component {
+  props: {
+    balance: Big,
+    symbol: string,
+    custom?: boolean,
+    onRemove: (symbol: string) => void
```

Comment:
will never return something. I am not strict about it, intent is to show that this component is not interested in return values from callback.